client/containers/Pub/PubHeader/TitleGroup.tsx

```diff
@@ -16,12 +16,14 @@ type Props = {
 
 const TitleGroup = (props: Props) => {
 	const { pubData, updatePubData } = props;
-	const { title, htmlTitle, description, isRelease, isReviewingPub } = pubData;
+	const { title, htmlTitle, description, isRelease, isReviewingPub, isAVisitingCommenter } =
+		pubData;
 	const { communityData, scopeData, featureFlags } = usePageContext();
 	const { submissionState } = usePubContext();
 	const isUnsubmitted = submissionState?.submission.status === 'incomplete';
 	const { canManage } = scopeData.activePermissions;
-	const canModify = canManage && !isRelease && !isUnsubmitted && !isReviewingPub;
+	const canModify =
```

Comment:
I don't think we need to modify this check here. If a user `canManage` it doesn't really matter whether `isReviewingPub` or `isAVisitingCommenter` — we still want to give them the ability to do it.